### PR TITLE
Describe the Layout Updates Options columns for the "Using a Widget to Position a Block" page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes the Layout Updates Options columns for the "Using a Widget to Position a Block" page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/cms/widget-static-block.html

### CODE REVIEW

File: src/cms/widget-static-block.md [hunk 1]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
    ![]({% link images/images/widget-layout-update-home-page.png %}){: .zoom}
    _Widget layout updates_
 
+### Layout Updates Options
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:55]:
```suggestion
### Layout update options
```

File: src/cms/widget-static-block.md [hunk 2]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
...
+
+{: .fields-table }
+|**_Categories_**||
+|Anchor Categories|Displays the widget on the anchor category page. Options:<br/>**Categories** - Categories where the anchor will be displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
+|Non-Anchor Categories|Displays the widget on the non-anchor category page. Options:<br/>**Categories** - Categories where the anchor will be displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:60]:
```suggestion
|Non-Anchor Categories|Displays the widget on the non-anchor category page. Options:<br/>**Categories** - Categories where the anchor is displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

File: src/cms/widget-static-block.md [hunk 3]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
    ![]({% link images/images/widget-layout-update-home-page.png %}){: .zoom}
    _Widget layout updates_
 
+### Layout Updates Options
+
+{: .fields-table }
+|**_Categories_**||
+|Anchor Categories|Displays the widget on the anchor category page. Options:<br/>**Categories** - Categories where the anchor will be displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:59]:
```suggestion
|Anchor Categories|Displays the widget on the anchor category page. Options:<br/>**Categories** - Categories where the anchor is displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

File: src/cms/widget-static-block.md [hunk 4]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
...
+|Non-Anchor Categories|Displays the widget on the non-anchor category page. Options:<br/>**Categories** - Categories where the anchor will be displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
+|**_Products_**||
+|All Product Types|Displays the widget on either a specific type of product page, or on all product pages. Options:<br/>**Products** - Products for which the widget is displayed. Options: All / Specific Products<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:62]:
```suggestion
|All Product Types|Displays the widget on either a specific type of product page, or on all product pages. Options:<br/>**Products** - Products for which the widget is displayed. Options: All / Specific Products<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

File: src/cms/widget-static-block.md [hunk 5]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
...
+|**_Generic Pages_**||
+|All Pages|Displays the widget on all pages. Options:<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:64]:
```suggestion
|All Pages|Displays the widget on all pages. Options:<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

File: src/cms/widget-static-block.md [hunk 6]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
...
+|Specified Page|Displays the widget on a specific page. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:65]:
```suggestion
|Specified Page|Displays the widget on a specific page. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

File: src/cms/widget-static-block.md [hunk 7]
```diff
@@ -52,6 +52,19 @@ _Widgets list_
...
+|Page Layouts|Displays the widget on pages with a certain layout. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|
```
Comment (leslietilling) [inline src/cms/widget-static-block.md:66]:
```suggestion
|Page Layouts|Displays the widget on pages with a certain layout. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
minor edits

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**